tests/tests_pytorch/conftest.py

```diff
@@ -190,6 +190,21 @@ def tmpdir_server(tmpdir):
         server.shutdown()
 
 
+@pytest.fixture
+def clean_import():
```

Comment:
Can you add a docstring about what this does?

Reply:
Done